imports/plugins/included/notifications/client/components/notificationRoute.js

```diff
@@ -43,14 +36,23 @@ class NotificationRoute extends Component {
     return markOneAsRead(notify._id);
   }
 
+  handleMarkAllAsRead = () => {
+    const { notificationList, markAllAsRead } = this.props;
+    markAllAsRead(notificationList);
+  }
+
   renderDropdownHead() {
-    const { notificationList, unread, markAllAsRead } = this.props;
+    const { unread } = this.props;
```

Comment:
@mikemurray you mentioned not to import with `{ }` if it's a single const for speed, should we turn this into just `const unread = this.props.unread;`?

Reply:
we do have a rule `prefer-destructuring` coming in #3573  that is to enforce doing `{ } ` everywhere, even for single const.

Reply:
@kieckhafer @impactmass Since the new rules dictate this new style, I'll make the necessary adjustments when #3573 hits master.

Reply:
sure, sounds good! I just thought to add 👍